helm/hpcc/templates/_helpers.tpl

```diff
@@ -236,7 +237,7 @@ Add data volume mount
 Pass in root
 */}}
 {{- define "hpcc.addDataVolumeMount" -}}
-{{- include "hpcc.addVolumeMounts" (dict "root" .root "includeLabels" (list "data" "")) -}}
+{{- include "hpcc.addVolumeMounts" (dict "root" .root "includeLabels" (list "data" "" "lz")) -}}
```

Comment:
This change means all engines will auto mount/expose lz planes.
Is that intended/needed/related to this PR?

Reply:
It was intended - so that they can use the plane::x:: syntax to directly read data from the landing zones - otherwise you would have to spray before you can read data from a lz.  I'm happy to debate whether this is a sensible change...

Reply:
ok, reasoning makes sense.
It may be more that we want to restrict access to LZ's by default, but by user authorization of planes.